### PR TITLE
find hash using git blame; closes #83

### LINKED ISSUE #83: Use git blame to find the hash of your file
In:
https://coderefinery.github.io/git-collaborative/01-remotes/
For 
10. Create a branch yourname/somefeature pointing at your commit

First find out the hash of your commit (using git graph or git log), then create a branch “in the past” pointing to that hash:

$ git branch yourname/somefeature [hash]

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: _episodes/01-remotes.md [hunk 1]
```diff
@@ -201,8 +201,14 @@ It will work for one more person.
 
 ### 10. Create a branch `yourname/somefeature` pointing at your commit
 
-First find out the hash of your commit (using `git graph` or `git
-log`), then create a branch "in the past" pointing to that hash:
+First find out the hash of your commit. You can do this using `git graph` or `git log`
+but another nice way of finding your commit is to use `git blame` on the file which you have added:
+
+```
+$ git blame yourusername.txt
```
Comment (rkdarst) [inline _episodes/01-remotes.md]:
```suggestion
$ git log yourusername.txt
```
See comment in #83 for why I suggest this.  The paragraph above would have to be adjusted too.

Comment (rkdarst) [reply _episodes/01-remotes.md]: Also then https://github.com/coderefinery/template-centralized-workflow-exercise/pull/1 needs updating.

Comment (bast) [reply _episodes/01-remotes.md]: I agree. Working on adjusting this ...

Comment (bast) [reply _episodes/01-remotes.md]: Done, both here and in the exercise repo.

**Pull request merged**